frontend/js/components/Repeater.vue

```diff
@@ -77,7 +77,22 @@
       blocks: {
         get () {
           if (this.savedBlocks.hasOwnProperty(this.name)) {
-            return this.savedBlocks[this.name] || []
+            const blocks = this.savedBlocks[this.name]
+            if (blocks && blocks.length > 0) {
+              const repeaterName = this.name.replace(/(blocks-\d+_)/, '')
+              const titleFieldToMap = this.$store.state.repeaters.availableRepeaters[repeaterName] && this.$store.state.repeaters.availableRepeaters[repeaterName].titleField ? this.$store.state.repeaters.availableRepeaters[repeaterName].titleField : null
+              if (titleFieldToMap) {
+                return blocks.map(block => {
+                  const titleField = this.$store.state.form.fields.find(field => field.name === `blocks[${block.id}][${titleFieldToMap}]`)
```

Comment:
I'm not very fan of that here. I'm thinking about performances issues with a repeater with a lot of blocks.

Currently, I see one different solution:
Pass `titleField` to map as a `prop` inside `Blocks.vue` component and move logic to get the value of field inside `Blocks.vue` component.

**Pros:**
* It will be update only one block and not try to update all blocks when a user update field value.
* We can use this behavior with content editor.

**Cons:**
* It will add a dependency to vuex store inside `Blocks.vue` component.